example/ios/Podfile

```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-platform :ios, '11.1'
+platform :ios, '14.0'
```

Comment:
When trying to run the example app but getting some errors with deployement version and I see some other places that appear to need to update the target to 14.

`square_reader_sdk.podspec`'s `s.ios.deployment_target`
Runner.xcodeproj/project.pbxproj is still referencing ios 11